src/io/json11.cpp

```diff
@@ -165,7 +165,7 @@ class Value : public JsonValue {
         return m_value == static_cast<const Value<tag, T> *>(other)->m_value;
     }
     bool less(const JsonValue * other) const override {
-        return m_value < static_cast<const Value<tag, T> *>(other)->m_value;
+        return m_value < (static_cast<const Value<tag, T> *>(other)->m_value);
```

Comment:
@guolinke the only thing I changed in your code was adding these parentheses. My editor was complaining because it interpreted that first `<` as the beginning of a cast instead of a logical operator 😆